prereqs.sh : Archive unused/retired scripts, and closes #978 , #976

rdlrt opened the pull request.

Scitz0 reviewed (approved):
👍 
Should we also "archive" the ITN rewards/metadata section in docs?

rdlrt commented:
> 👍
> Should we also "archive" the ITN rewards/metadata section in docs?

Updated, I've kept the pages (so links would still work), just removed references from sidebar
